addons/account/views/report_payment_receipt_templates.xml

```diff
@@ -52,22 +53,28 @@
                                     <td><span t-field="inv.invoice_date"/></td>
                                     <td><span t-field="inv.name"/></td>
                                     <td><span t-field="inv.ref"/></td>
+                                    <td/>
                                     <td class="text-right"><span t-field="inv.amount_total"/></td>
                                 </tr>
                                 <!-- PAYMENTS/REVERSALS -->
                                 <tr t-foreach="inv._get_reconciled_invoices_partials()" t-as="par">
-                                    <t t-set="amount" t-value="par[1]"/>
+                                    <t t-set="amountPayment" t-value="par[2].amount_currency"/>
+                                    <t t-set="currencyPayment" t-value="par[0].credit_currency_id"/>
```

Comment:
`credit_currency_id` is the payment currency in invoices, but not in bills
We need a standard way of retrieving the currency i.e
```
<t t-set="currencyPayment" t-value="par[2].currency_id"/>
<t t-set="currencyInvoice" t-value="inv.currency_id"/>
```